### PR TITLE
[ProxyManagerBridge] Adding info about discontinuation

### PR DESCRIPTION
**Pull request by ThomasLandauer**
| Q             | A
| ------------- | ---
| Branch?       | 6.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | 
| License       | MIT

### CODE REVIEW

File: src/Symfony/Bridge/ProxyManager/README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ ProxyManager Bridge
 
 The ProxyManager bridge provides integration for [ProxyManager][1] with various
 Symfony components.
+The project is discontinued; 6.4 is the last version.
```
Comment (nicolas-grekas) [inline src/Symfony/Bridge/ProxyManager/README.md]: let's add a blank line before to make  a new §

Comment (ThomasLandauer) [reply src/Symfony/Bridge/ProxyManager/README.md]: Sure, I made it bold too.

File: src/Symfony/Bridge/ProxyManager/README.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ ProxyManager Bridge
 The ProxyManager bridge provides integration for [ProxyManager][1] with various
 Symfony components.
 
+**The project is discontinued; 6.4 is the last version.**
```
Comment (stloyd) [inline src/Symfony/Bridge/ProxyManager/README.md]: Maybe use GH markdown?
```suggestion
> [!WARNING]  
> The project is discontinued; 6.4 is the last version.
```

**Review by GromNaN — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @ThomasLandauer.

**Comment by ThomasLandauer** [comment]
@nicolas-grekas Looks like you need to tag a new version, to actually get this online at https://github.com/symfony/proxy-manager-bridge ;-)

**Comment by xabbuh** [comment]
New patch releases are done usually once per month.

**Comment by nicolas-grekas** [comment]
The subtree-split was out of sync, this is now fixed.

**Comment by alexander-schranz** [comment]
Short question here what is the first version of Symfony not longer require the proxy-manager-bridge for lazy services? Is it 6.4?

**Comment by nicolas-grekas** [comment]
6.4 yes

**Comment by nicolas-grekas** [comment]
6.2 actually!

**Pull request merged**